libraries/cmake/source/augeas/README.md

```diff
@@ -61,55 +62,49 @@ Copy the generated gnulib files:
  - `config.h` (use the augeas one)
  - All the files containing `AUTOGENERATED` under `lib`
 
-# macOS
+## macOS
```

Comment:
nit: Why two # here (and the additional # below)? This not a sub-heading of Linux

Reply:
In the Markdown files I edited that had more than one top-level heading I was trying to fix them but I may have made some mistakes, I'll go through again (thanks for spotting)